globus_automate_client/cli/auth.py

```diff
@@ -68,16 +72,16 @@ def __init__(self, token_store: Union[pathlib.Path, str]):
         self.modified = False
 
     @property
-    def tokens_for_environment(self) -> Dict[str, TokenSet]:
+    def tokens_for_environment(self):
         """
         We will sub-key the full token set for environments other than production
         """
         environ = os.environ.get("GLOBUS_SDK_ENVIRONMENT")
-        if environ in {None, "production", "prod", "default"}:
+        if environ is None or environ in {"production", "prod", "default"}:
             return self.tokens
         environ_cache_key = TokenCache._environment_prefix + environ
         if environ_cache_key not in self.tokens:
-            self.tokens[environ_cache_key]: Dict[str, TokenSet] = {}
+            self.tokens[environ_cache_key] = {}
```

Comment:
The annotation here is effectively (useful) documentation, does a linter complain about this?

Reply:
My memory is that mypy didn't complain but PyCharm highlighted this as extraneous. It's annotating a dictionary key's value instead of the dictionary itself so it didn't seem to add additional value. Should I revert this?

Reply:
I'd vote in favor of keeping because it made it easier to understand what `self.tokens` is doing.

Reply:
I'll get this fixed.

Separately, the token cache annotations triggered a number of errors throughout the codebase with mypy. Eventually this needs to be addressed better than what I did in 36f84f1026483cbb7b6d249b8b50fde114eec5c8.